tests/tests/tier0/bluechictl-monitor-unit-created-removed/python/monitor.py

```diff
@@ -125,8 +108,7 @@ def test_monitor_specific_node_and_unit(self):
         failsafe_thread.start()
 
         # Running bluechictl status on inactive unit should raise UnitCreated and UnitRemoved signals
-        res, out, _ = self.run_command(f"bluechictl status {node_name_foo} {service_simple}")
-        assert res == 0
+        out = Command(f"bluechictl status {node_name_foo} {service_simple}").run()
```

Comment:
Why not just use `bluechi_machine_lib.bluechictl.BluechiCtl`?

Reply:
Though having this module in `bluechi_machine_lib` does make a lot of sense, I would actually change BluechiCtl to use it

Reply:
Yes, we have some duplication already with bluechictl. We need to fine-tune this a bit. But thats the focus of another PR :grin: